testng-core/src/test/java/test/listeners/github1296/MyConfigurationListener.java

```diff
@@ -1,23 +1,30 @@
 package test.listeners.github1296;
 
-import java.util.HashMap;
 import java.util.Map;
+import java.util.Map.Entry;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
 import org.testng.IConfigurationListener;
 import org.testng.ITestResult;
 
 public class MyConfigurationListener implements IConfigurationListener {
 
-  public static final Map<String, Integer> CALLS = new HashMap<>();
+  private static final Map<String, AtomicInteger> CALLS = new ConcurrentHashMap<>();
+
+  public MyConfigurationListener() {
+    CALLS.clear();
```

Comment:
Please clear data from the test because it doesn't make sense that the tested object is modifying the way it is tested.

Reply:
fixed